apps/api/src/app/integrations/usecases/get-active-integration/get-active-integration.usecase.ts

```diff
@@ -16,6 +16,7 @@ export class GetActiveIntegrations {
         organizationId: command.organizationId,
         environmentId: command.environmentId,
         userId: command.userId,
+        filterByEnvironment: command.filterByEnvironment,
```

Comment:
Currently, the usecase used to return active active integrations of all environments
this will allow to filter and get active integrations for the current env